### ISSUE TITLE
Add brotli size computation

### ISSUE THREAD

**Issue by rifler** [ai/size-limit#144]
about 90% of browsers support brotli encoding
https://caniuse.com/#search=brotli

**Comment by ai** [comment]
1. What use case do you have to compare JS by Brotli?
2. What API do you suggest?

**Comment by rifler** [comment]
1. I don't understand your question
2.
a) we can remove gzip and show only brotli
b) show both sizes:
```
  Size (gzip):           20.52 KB with all dependencies, minified and compressed
  Size (brotli):         18.76 KB with all dependencies, minified and compressed
  Loading time (gzip):   411 ms   on slow 3G
  Loading time (brolti): 395 ms   on slow 3G
  Running time:          80 ms    on Snapdragon 410
  Total time (gzip):     490 ms
  Total time (brotli):   470 ms
```
But it looks little bit noisy. We can play with display format

**Comment by ai** [comment]
Remove gzip and show only Brotli is a good idea

**Comment by ai** [comment]
Do you want to prepare PR? But we will need an option to show gzip numbers as well.

**Comment by rifler** [comment]
I'll try soon

**Comment by binjospookie** [comment]
`createBrotliCompress` has been added in: v11.7.0. I found that size-limit's CI config file (.travis.yml) includes Node.js versions below (8, 10). 
Does it mean deprecation versions below 11.7.0 or just showing a message like "Brotli report will be available after upgrading node above v11.7.0)?

**Comment by ai** [comment]
Warning will be enough (since it is opt-in feature)

**Comment by ai** [comment]
Released in 3.0
